apps/web/src/components/Basenames/UsernameProfileEditModal/index.tsx

```diff
@@ -244,6 +244,12 @@ export default function UsernameProfileEditModal({
             value={textRecords[UsernameTextRecordKeys.Description]}
             disabled={isLoading}
           />
+          <UsernameTextRecordInlineField
+            textRecordKey={UsernameTextRecordKeys.Frame}
+            onChange={onChangeTextRecord}
+            value={textRecords[UsernameTextRecordKeys.Frame]}
+            disabled={isLoading}
+          />
```

Comment:
since this is the minimal implementation and we're not sure if we're going to be able to do full interactivity yet with wallet signer or xmtp, I've just added a text field to the edit profile modal